config.go

```diff
@@ -78,6 +79,22 @@ func (c *Config) exists() (bool, error) {
 	return afero.Exists(Fs, c.filePath)
 }
 
+func updateConfNameAndGroup(newdevice string, newgroup string, filePath string) error {
+	conf, err := GetConfig(filePath)
+	if err != nil {
+		return err
+	}
+	conf.DeviceName = newdevice
+	conf.Group = newgroup
+	Fs.Remove(filePath)
```

Comment:
There should be no reason to remove the file before calling `write()`. It'll overwrite it.